backend/server/models.py

```diff
@@ -42,12 +42,12 @@ class Meeting(models.Model):
     # contributer - people who opened the meeting with the host
     max_participant = models.IntegerField()
     deadline = models.DateTimeField('meeting deadline')
-    region = models.CharField(max_length=100, default="")
-    photo = models.ImageField(upload_to=pic_folder, default=DEFAULT_IMAGE)
+    region = models.CharField(max_length=100, blank=True)
```

Comment:
please add two float field, (latitude and longitude)
this Charfield will be used for name of place. (i.e. 과방)

Reply:
So, our service automatically allocate region, or host type the region?

And region is based on '구'? or something else? maybe we should decide and make some guidelines for region field.
It would be quite hard to search all the exact location and calculate the distances, and sort, ...

Reply:
Kakao map API provides function that can translate LatLng to exact address(서울특별시 관악구 관악로 1). So we do not have to keep this address, but we have to keep building name or address name etc(like 301동 314호)